### PR TITLE
Document ZooKeeper credentials. Add missing examples.

### PR DESCRIPTION
**Pull request by ghost**
## Description
https://jira.mesosphere.com/browse/DOCS-2131

## Urgency
- [ ] Blocker 
- [x ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, and 1.10).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by vespian** [comment]
@adamtheturtle Could you please also have a look? I can't add you as a reviewer :(

File: 1.10/installing/custom/configuration/examples.md
Comment (vespian) [inline 1.10/installing/custom/configuration/examples.md]: Should we explicitly say here that this is a string?

**Comment by stbof** [comment]
@vespian  Joel left Mesosphere last summer.

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (vespian) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: Can we please rephrase it somehow?

I would like to suggest: `Credentials used by bootstrapping code to access the credentials of the services that will be running on the agents.`

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (vespian) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: Same as below - can we please mention here that these creds will be used by the bootstraping code?

**Review by vespian — CHANGES_REQUESTED** [review]
Left some comments.

**Comment by vespian** [comment]
> @vespian Joel left Mesosphere last summer.
@stbof - @adamtheturtle is the GH handle of the member of the security team. He is very good at writing technical documents and I would like to get his opinion on this.

**Review by gpaul — APPROVED** [review]

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: FTR I am just working on the assumption that these links are done in the right way.

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: > In `strict` and `permissive` clusters

From the rest of the documentation, we say things like:

"On DC/OS clusters in strict and permissive mode...".

Comment (adamtheturtle) [reply 1.10/installing/custom/configuration/configuration-parameters.md]: The first time we use ACL in a section let's use it like we do in some of our other documentation:

> access control lists (ACLs)

Comment (adamtheturtle) [reply 1.10/installing/custom/configuration/configuration-parameters.md]: > protected by ACLs

ACLs don't protect anything.
Instead, perhaps "protected using ACLs"?

Comment (adamtheturtle) [reply 1.10/installing/custom/configuration/configuration-parameters.md]: > can't

Let's use "cannot" and ideally this would be in a style guide. For now it is my personal, internal style guide that I'm happy to discuss.

Comment (adamtheturtle) [reply 1.10/installing/custom/configuration/configuration-parameters.md]: > ZooKeeper quorum 

I believe that this should be "the ZooKeeper Quorum".

Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: > ACLs specify sets of IDs and permissions that are associated with those IDs

Rather than "IDs and permissions" I believe that these are resource IDs (RIDs) and actions".

Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: > `world`, `auth`, `digest`, `host`, `ip`. 

Let's put "and" before the last item in this list.

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: > use `digest` authentication, which require a [...] string

This isn't grammatically correct. If the meaning is that the authentication itself requires a string, change "require" to "requires".

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: > equivalent of `root/superuser` account

either "of a" or "of the".

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: Can this have some highlighting? Perhaps:

````
```console
...
```
````

?

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: > DC/OS Master nodes

The rest of our documentation seems to use lowercase "m" for "master". e.g. "Admin Router runs on DC/OS master nodes".

Comment (adamtheturtle) [reply 1.10/installing/custom/configuration/configuration-parameters.md]: Change `zkCli` to `` `zkCli` ``.

Comment (adamtheturtle) [reply 1.10/installing/custom/configuration/configuration-parameters.md]: > the nodes in `/dcos` tree

This is not grammatically correct. I guess change "in" to "in the"?

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: ❓ Is it relevant that it is code? We don't say "code" elsewhere that I can find to refer to our own processes. Perhaps "used by the bootstrap process", but I'm not sure.

> the masters

Let's be clear and say "DC/OS master nodes" or whatever the right distinction is.

**Review by adamtheturtle — CHANGES_REQUESTED** [review]
Thank you @stbof . Please resubmit after the requested changes have been made, and feel free to ask the security team for help on specifics 😄

**Comment by stbof** [comment]
@adamtheturtle  I believe I have addressed all your feedback.  Our current web site doesn't support highlighting of console output, but our new doc website will.

**Review by vespian — APPROVED** [review]

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: I believe "quorum" should be "Quorum".
I'm sure that it should be consistent within this document.

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: Is this definitely powershell?

Comment (adamtheturtle) [reply 1.10/installing/custom/configuration/configuration-parameters.md]: I'm no PowerShell expert, but I think that it isn't, as this includes `ls` when PowerShell uses `dir`. Instead, this is `console`.

The commands alone are either `bash` or `sh`, but as this shows the prompt, it is `console` which is generic.

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: Missing article before "bootstrapping".

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: Missing article before "bootstrapping" - I guess "the bootstrapping process" but @vespian might know the used term better than I do.

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: This is now "DC/OS Enterprise" and not "Enterprise DC/OS" I believe.

Comment (stbof) [reply 1.10/installing/custom/configuration/configuration-parameters.md]: @adamtheturtle Yes, but we are not doing this wholesale renaming until we deploy the new merged DC/OS Enterprise and DC/OS open source doc sites.

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (adamtheturtle) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: This implies that there is an account called "root/superuser" at least in some system. Instead, some systems use "root", some use "superuser" and this account is equivalent to both.

**Review by adamtheturtle — CHANGES_REQUESTED** [review]
Thanks @stbof - I've spotted all that I will today 😄

File: 1.10/installing/custom/configuration/configuration-parameters.md
Comment (vespian) [reply 1.10/installing/custom/configuration/configuration-parameters.md]: `the bootstrapping processes` please :)

**Pull request merged**
